src/components/editor/editor-pane/upload-handler.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Editor } from 'codemirror'
+import i18n from 'i18next'
+import { uploadFile } from '../../../api/media'
+import { store } from '../../../redux'
+
+export const handleUpload = (files: FileList | null, editor: Editor | undefined): void => {
```

Comment:
Why are you allowing empty values for the parameters?

Reply:
To easily pass the parameter from the events / react context and handle null / undefined in this handler instead of on multiple places.